doc/src/odin/dev_guide/contributing.md

```diff
@@ -18,12 +18,12 @@ Add a description of the changes made and reference the "issue" that it corrects
 
 ### Creating a "Work in progress" (WIP) PR
 
-**Important** Before creating a WIP PR, if it is a bug you have happened upon and intend to fix make sure you create an issue beforhand.
+**Important** Before creating a WIP PR, if it is a bug you have happened upon and intend to fix make sure you create an issue beforehand.
 
 A "work in progress" PR is a pull request that isn't complete or ready to be merged.
 It is intended to demonstrate that an Issue is being addressed and indicates to other developpers that they don't need to fix it.
```

Comment:
developers spelling - too minor to create a new PR; if you can handle in this one itself!